swo_aws_extension/extension.py

```
@@ -36,10 +38,19 @@ def process_order_fulfillment(client, event):
     "/v1/orders/validate",
     response={
         HTTPStatus.OK: dict,
-        HTTPStatus.BAD_REQUEST: Error,
+        HTTPStatus.BAD_REQUEST: dict,
     },
     auth=JWTAuth(jwt_secret_callback),
 )
-def process_order_validation(request, order: Annotated[dict | None, Body()] = None):
+def process_order_validation(request, order: Annotated[dict, Body()]):
```

Ruff (0.14.10)

Unused function argument: `request`

(ARG001)